platform_versioned_docs/version-24.2/enterprise/advanced-topics/seqera-container-images.mdx

```diff
@@ -53,11 +53,11 @@ The Seqera container registry `cr.seqera.io` is the default Tower container imag
                 ```
 
     === "Azure"
-        Seqera publishes legacy Tower Enterprise containers to a private Azure Container Registry instance. Retrieve them with the following steps:
+        Seqera publishes legacy Seqera Enterprise containers to a private Azure Container Registry instance. Retrieve them with the following steps:
```

Comment:
Let me check if we still upload images to GCP and Azure... we don't touch those clouds often

Reply:
Thanks! I was about to ask you in Slack whether those even still exist for customers

Reply:
I can strip Azure and GCP out of the 24.2 and 24.3 versions of this page if needed.

Reply:
slack thread where I ask more details: https://seqera.slack.com/archives/C01GNJ63HLY/p1741869656888429

TBH I don't think those registries exist anymore though: last mention was from you last year asking whether they still exist, before then it was in 2022.